bin/deploy.sh

```diff
@@ -59,6 +59,7 @@ done
 
 if [ "$PUBLISH" == 0 ]; then
     if [ ! -r "$FILE" ]; then echo "Error: Cannot read file '$FILE'. Try '$0 -h'" && exit 1; fi
+    openstack image delete "$IMG_NAME - PR#$PR_NUM"
```

Comment:
Ideally you should be using the UUID as you may have multiple images with the same name, especially with how it currently works. It might be better to list the images, and then grep for ones matching this and grab the UUID's.